### PR TITLE
finish abs path refactoring

### PR DESCRIPTION
**Pull request by jamesbeedy**

### CODE REVIEW

File: lib/tlslib.py [hunk 1]
```diff
@@ -141,8 +149,11 @@ def ca(source, destination, user=None, group=None):
 
     if not source:
         # When source not specified use the default CA path.
-        source = '/usr/local/share/ca-certificates/{0}.crt'.format(
-            hookenv.service_name())
+        source = \
```
Comment (mbruzek) [inline lib/tlslib.py]: This source is already an absolute path and is not based on 'charm_dir' it is the location where ca-certificates are installed.

**Comment by mbruzek** [comment]
@jamesbeedy Thanks for following through with this pull request. I am embarrassed that I missed the library code! Overall this looks like a good change with the exception of the last path which was already an absolute path and should not be based on the charm_dir. If you remove that I can merge this PR.

File: lib/tlslib.py (hunk 1 above)
Comment (jamesbeedy) [inline lib/tlslib.py]: @mbruzek, good eye!

**Comment by mbruzek** [comment]
+1 Looks good to me. Thank you for the contribution James.

**Pull request merged**